elastalert/alerts.py

```diff
@@ -34,7 +35,8 @@ def _add_custom_alert_text(self):
         alert_text = self.rule.get('alert_text', '')
         if 'alert_text_args' in self.rule:
             alert_text_args = self.rule.get('alert_text_args')
```

Comment:
not from this CR, but .get('alert_text_args', []) or otherwise the next lines will raise